ghost/i18n/locales/fr/comments.json

```diff
@@ -33,11 +33,11 @@
     "Head of Marketing at Acme, Inc": "Responsable du marketing chez Acme, Inc",
     "Hide": "Masquer",
     "Hide comment": "Masquer le commentaire",
-    "Jamie Larson": "Jamie Larson",
-    "Join the discussion": "Rejoignez la discussion",
+    "Jamie Larson": "Jean Martin",
+    "Join the discussion": "Rejoindre la discussion",
     "Just now": "À l'instant",
     "Local resident": "Résident local",
-    "Member discussion": "Discussion entre membres",
+    "Member discussion": "Discussion entre lecteurs",
```

Comment:
I'd stick to `abonnés` here, as we usually don't use the term readers in English either, only members. The word `abonné` works for both free and paid members

Reply:
Agree that the initial "membres" does not work well

Reply:
Okay